<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>492</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MD5快速计算器（v1.31 α）</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/logo/icon.png</normaloff>:/logo/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
color:rgb(250, 250, 250);
font: 9pt &quot;微软雅黑&quot;;
}
QPushButton{
color:rgb(250, 250, 250);
border:1px solid rgb(128, 128, 128);
background-color: rgb(40 ,40 ,40);
width: 60px;
height: 30px;
border-radius:5px;
}
QPushButton:hover{
border-color:rgb(255, 170, 0);
}
QPushButton:pressed{
background-color: rgb(255, 170, 0);
border-color: rgb(128, 128, 128);
color:black;
}
QPushButton:disabled{
background-color: rgb(60, 60, 60);
color: rgb(160, 160, 160);
}
QTextEdit, QLineEdit{
background-color: rgb(40 ,40 ,40);
border:1px solid rgb(128, 128, 128);
border-radius:5px;
}
#MainWindow, #CompareWnd, QDialog{
background-color: rgb(80, 80, 80);
}
#stcResult{
color:rgb(250, 250, 250);
font: 9pt &quot;微软雅黑&quot;;
font-weight: bold;
}
QListWidget{
background-color: rgb(40, 40, 40);
border-radius:5px;
}
QListWidget::item:hover{
border: none;
background: none;
}
QListWidget::item:selected{
color: rgb(40,40,40);
background-color: rgb(255, 170, 0);
border: none;
border-radius:5px;
}
QProgressBar{
	font:9pt;
	border-radius:5px;
	text-align:center;
	border:1px solid #808080;
	background-color: rgb(40, 40, 40);
}
QProgressBar:chunk{
	border-radius:5px;
	background-color:rgb(255, 170, 0);
}</string>
  </property>
  <widget class="QPushButton" name="btnOpenFile">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加文件...</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lnkClear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;计算结果：（&lt;a href=&quot;javascript:void(0)&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#ffaa00;&quot;&gt;点此清空&lt;/span&gt;&lt;/a&gt;）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtMd5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalc">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算当前文件MD5</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCompare">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>校验MD5...</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="stcAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MD5快速计算器 v1.31 (Alpha 2021/08/12) by szx0427（基于 Qt 4.8.7）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前进度：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progMd5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveResult">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存结果...</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCopyMd5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>复制当前MD5</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>931</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::MoveAction</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelCur">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除当前选择的项目</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelAll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清除全部项目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>要计算的文件列表：</string>
   </property>
  </widget>
  <widget class="QLabel" name="stcCount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已添加 0 个文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalcAll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算全部文件MD5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>总体进度：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progAll">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可拖拽更改项目顺序</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
